turbo.json

```diff
@@ -10,51 +10,57 @@
         "generated/spec.schema.json"
       ]
     },
-    "generate:typescript": {
+    "@realm/bindgen#build": {
       "inputs": [
-        "src/*",
-        "spec.yml"
+        "src/*"
+      ],
+      "dependsOn": [
+        "generate:spec-schema"
+      ]
+    },
+    "@realm/binding#compile": {
+      "inputs": [
+        "generated/node/*"
       ],
       "outputs": [
-        "generated/ts/*"
+        "build/Release/*",
+        "build/Debug/*"
       ],
       "dependsOn": [
-        "generate:spec-schema"
+        "^build"
       ]
     },
-    "generate:node": {
+    "@realm/binding#build": {
       "inputs": [
-        "src/*",
-        "spec.yml"
+        "src/*"
```

Comment:
Don't you also need generated/* in here?

Reply:
The task depends on `generate:spec-schema` which declares this as an output, so I'd expect the "generated/spec.schema.json" to either be rebuild or restored from cache correctly.

But .. I'll throw it in for good measures. Let's see how it works and tweak it if we find that problematic.